addons/udes_stock/tests/test_stock_quant.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+
+from . import common
+from datetime import datetime, timedelta
+
+DATE_FMT = "%Y-%m-%d %H:%M:%S"
```

Comment:
Nit (feel free to ignore): to avoid hard coding this you could instead get the format from odoo
`from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT`

Reply:
Noice - done